docs/GroupDefs.hpp

```diff
@@ -453,6 +453,35 @@
  * \brief Tags used for options parsed from the input file
  */
 
+/*!
+ * \defgroup LinearSolverGroup  Linear Solver
+ * \brief Algorithms to solve linear systems of equations
+ *
+ * \details In a way, the linear solver is for elliptic systems what time
+ * stepping is for the evolution code. This is because the DG scheme for an
+ * elliptic system reduces to a linear system of equations of the type
+ * \f$Ax=b\f$, where \f$A\f$ is a global matrix representing the DG
+ * discretization of the problem. Since this is one equation for each node in
+ * the computational domain it becomes unfeasible to numerically invert the
+ * global matrix \f$A\f$. Instead, we solve the problem iteratively so that we
+ * never need to construct \f$A\f$ globally but only need the projection
+ * \f$Ax\f$ that can be evaluated locally by virtue of the DG formulation. This
```

Comment:
This may just be a terminology difference between fields, but I am confused by this being referred to as a projection, since in the linear algebra sense "Ax" is not a projection of either "A" or "x".

Reply:
Agreed. Do you have a suggestion for a better name? I mean to say "The linear operator applied to the `Tag`".

Reply:
The tag should also be renamed, but at the moment I'm confused by the documentation, since I'm not sure what "Ax" refers to here.  Is it the global A times the global x, some local A times the local x?  Is it the first of those restricted to the element?  Something else?

Reply:
Yes, the first of those restricted to the element. That’s what I mean with the comment that it can be evaluated locally.

Reply:
In terms of names, all that I can think of is along the lines of:
- `LinearSolver::Tags::op<Tag>`
- `LinearSolver::Tags::Operator<Tag>`
- `LinearSolver::Tags::OperatorValue<Tag>`
- `LinearSolver::Tags::OperatorAppliedTo<Tag>`
Perhaps `Operator<Tag>` is fine because it closely matches `A(x)`?